build-support/bin/bootstrap_pants_pex.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../.. && pwd -P)"
 
+cd "$REPO_ROOT" || exit 1
```

Comment:
Could be a separate change. Would it more make sense to `set -e` at the top instead of ` || exit 1` in a few places?